kickstart/gulpfile.js

```diff
@@ -16,64 +16,56 @@
 
 const gulp = require('gulp-help')(require('gulp'));
 const del = require('del');
+const mustache = require('gulp-mustache');
 const posthtml = require('gulp-posthtml');
 const postcss = require('gulp-postcss');
 const runSequence = require('run-sequence');
 
+function getData() {
+  return JSON.parse(require('fs').readFileSync('data.json'));
+}
+
```

Comment:
currently its in one file and we just namespace it (which is why the template has to do {{index.message}} instead of just {{message}}). I'll see if we can split it up in a follow up PR